packages/esm-patient-orders-app/src/order-basket/order-basket.workspace.tsx

```diff
@@ -116,21 +124,27 @@ const OrderBasket: React.FC<DefaultWorkspaceProps> = ({
             <InlineNotification
               kind="error"
               title={t('errorCreatingAnEncounter', 'Error when creating an encounter')}
-              subtitle={t('tryReopeningTheWorkspaceAgain', 'Please try launching the workspace again')}
```

Comment:
I think it's best to show the original message here instead of the error message from the backend, which we're already logging to the console on line 67.

Reply:
I think that's right. 
Just what I thought that given on creating the encounter we're  adding orders too, then if the error was from the orders, showing the exact message might be helpful to the user what exactly is the issue.

Reply:
Dennis may be the leading authority on UI copy; I would defer to him @usamaidrsk .